Plugin project - path to base commands are not being properly constructed

Attempted to upgrade a client project to `2.0.0-beta.1` and ran into this issue when running base commands:

```
bscot@BSCOTT-PC ~/proj/ (trunk) -> proj-cli -V
2.0.0-beta.1

bscot@BSCOTT-PC ~/proj/ (trunk) -> proj-cli deploy
module.js:550
    throw err;
    ^

Error: Cannot find module 'C:\Users\bscot\proj\node_modules\and-cli\and-cli-deploy.js'
    at Function.Module._resolveFilename (module.js:548:15)
    at Function.Module._load (module.js:475:25)
    at Function.Module.runMain (module.js:694:10)
    at startup (bootstrap_node.js:204:16)
    at bootstrap_node.js:625:3

bscot@BSCOTT-PC ~/proj/ (trunk) -> proj-cli dotnet
module.js:550
    throw err;
    ^

Error: Cannot find module 'C:\Users\bscot\proj\node_modules\and-cli\and-cli-dotnet.js'
    at Function.Module._resolveFilename (module.js:548:15)
    at Function.Module._load (module.js:475:25)
    at Function.Module.runMain (module.js:694:10)
    at startup (bootstrap_node.js:204:16)
    at bootstrap_node.js:625:3
```

I believe this was just a missed detail in the `CommandRegistry` module. There's a function that determines how to register the command based on whether the module is being imported or not (ie, it is being run as a plugin CLI). Since we are pushing out compiled `dist` folder, the `executableFile` path probably just needs to be prefixed with `dist/`

--- a/src/modules/command-registry.test.ts
+++ b/src/modules/command-registry.test.ts
@@ -547,11 +547,12 @@ describe("CommandRegistry", () => {
                     const { command, description } = faker.random.arrayElement(
                         CommandDefinitions
                     );
-                    // The expected file path should point to ./node_modules/and-cli/and-cli-{commandName}.js
+                    // The expected file path should point to the transpiled JS in the node_modules folder
                     const expectedFilePath = upath.join(
                         ".",
                         Constants.NODE_MODULES,
                         Constants.CLI_NAME,
+                        Constants.DIST,
                         `${Constants.CLI_NAME}-${command}.js`
                     );
                     CommandRegistry.initialize(true); // This is the important setup

--- a/src/modules/command-registry.ts
+++ b/src/modules/command-registry.ts
@@ -387,11 +387,12 @@ const _getAliasCommandDefinitions = (): CommandDefinition[] => {
 const _getExecutablePath = (commandName: string): string => {
     const filename = `${Constants.CLI_NAME}-${commandName}.js`;
     if (_isImportedModule != null && _isImportedModule) {
-        // Returns './node_modules/and-cli/and-cli-dotnet.js, for example
+        // Returns './node_modules/and-cli/dist/and-cli-dotnet.js', for example
         return upath.join(
             ".",
             Constants.NODE_MODULES,
             Constants.CLI_NAME,
+            Constants.DIST,
             filename
         );
     }

